static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -242,13 +243,13 @@ export const runAndTrackAbTests = (): Promise<void> => {
     return getAsyncTestsToRun().then(tests => {
         tests.forEach(test => test.variantToRun.test(test));
 
-        registerImpressionEvents(tests);
-        registerCompleteEvents(tests);
+        registerImpressionEvents(tests); // only if variants not function
```

Comment:
I'm a bit confused by the comments here, the behaviour hasn't changed right?

Reply:
Ah sorry, that was a 'nic is exploring the code' comment that I meant to remove.